src/main/java/seedu/recipe/model/ModelManager.java

```diff
@@ -142,6 +144,20 @@ public ObservableList<Recipe> getFilteredRecipeList() {
         return filteredRecipes;
     }
 
+    /**
+     * Set the current predicate on the filtered list
```

Comment:
Right, so now filter will only be applied on the current list and not the original list, correct?